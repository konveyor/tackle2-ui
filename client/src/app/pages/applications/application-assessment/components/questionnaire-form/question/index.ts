export { Question } from "./question";
export { QuestionHeader } from "./question-header";
export { QuestionBody } from "./question-body";
