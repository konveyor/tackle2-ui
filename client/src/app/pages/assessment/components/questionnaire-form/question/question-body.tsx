import React from "react";
import { StackItem } from "@patternfly/react-core";

export interface QuestionBodyProps {
  children?: React.ReactNode;
}

export const QuestionBody: React.FC<QuestionBodyProps> = ({
  children = null,
}) => {
  return <StackItem>{children}</StackItem>;
};
