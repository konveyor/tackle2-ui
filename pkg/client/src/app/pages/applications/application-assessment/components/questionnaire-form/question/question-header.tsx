import React from "react";
import { StackItem } from "@patternfly/react-core";

export interface QuestionHeaderProps {
  children?: React.ReactNode;
}

export const QuestionHeader: React.FC<QuestionHeaderProps> = ({
  children = null,
}) => {
  return <StackItem className="pf-u-pb-sm">{children}</StackItem>;
};
