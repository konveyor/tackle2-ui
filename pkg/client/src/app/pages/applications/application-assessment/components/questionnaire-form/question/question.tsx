import React from "react";
import { Stack } from "@patternfly/react-core";

export interface RadioButtonQuestionProps {
  children?: React.ReactNode;
}

export const Question: React.FC<RadioButtonQuestionProps> = ({
  children = null,
  ...rest
}) => {
  return <Stack {...rest}>{children}</Stack>;
};
