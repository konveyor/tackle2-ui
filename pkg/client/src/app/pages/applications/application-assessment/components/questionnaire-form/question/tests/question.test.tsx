import React from "react";
import { shallow } from "enzyme";
import { Question } from "../question";

describe("Question", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<Question>content</Question>);
    expect(wrapper).toMatchSnapshot();
  });
});
