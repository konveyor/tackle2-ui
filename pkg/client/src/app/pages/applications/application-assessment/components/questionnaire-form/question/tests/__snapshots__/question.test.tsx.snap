// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question Renders without crashing 1`] = `
<Stack>
  content
</Stack>
`;
